lib/core/managers/course_repository.dart

```diff
@@ -381,8 +378,7 @@ class CourseRepository {
     // If there isn't the grade yet, will fetch the summary.
     // We don't do this for every course to avoid losing time.
     for (final course in fetchedCourses) {
-      course.evaluation =
-          _getEvaluationForCourse(course, fetchedCoursesEvaluations);
+      course.evaluation = _getReviewForCourse(course, fetchedCourseReviews);
```

Comment:
This might be too picky but ideally you should rename `course.evaluation` as `course.review` 😅 
You can just ignore my comment if you think it's not worth it, as it might involve some more painful refactoring 😅

Reply:
Noo nice catch I'll do that, I thought I refactored them all hahaha

Reply:
Great then 😄